pages/wizard/openstack/configure_overcloud.py

```diff
@@ -0,0 +1,159 @@
+    _floating_ip_net_loc = (By.XPATH, "//input[@id = 'osp-floating-network']")
+    _floating_ip_net_gateway_loc = (By.XPATH, "//input[@id = 'osp-float-gatewway']")
+    _admin_password_loc = (By.XPATH, "//input[@id = 'osp_overcloud_password']")
+    _confirm_admin_password_loc = (By.XPATH, "//input[@id = 'confirm_osp_overcloud_password']")
+    _external_ceph_storage_loc = (By.XPATH, "//input[@id = 'ospCephStorageCheckbox']")
+
+    # Ceph storage locators
+    _ceph_external_mon_host_loc = (By.XPATH, "//input[@id = 'cephExternalMonHostInput']")
+    _ceph_cluster_fsid_loc = (By.XPATH, "//input[@id = 'cephClusterFSIDInput']")
+    _ceph_client_username_loc = (By.XPATH, "//input[@id = 'cephClientUsernameInput']")
+    _ceph_client_key_loc = (By.XPATH, "//input[@id = 'cephClientKeyInput']")
+    _nova_rbd_pool_name_loc = (By.XPATH, "//input[@id = 'novarRbdPoolNameInput']")
+    _cinder_rbd_pool_name_loc = (By.XPATH, "//input[@id = 'cinderRbdPoolNameInput']")
+    _glance_rbd_pool_name_loc = (By.XPATH, "//input[@id = 'glanceRbdPoolNameInput']")
+
+    ##############
+    # properties #
+    ##############
+    @property
+    def external_net_interface(self):
+        return self.selenium.find_element(*self._external_net_interface_loc)
+
+    @property
+    def private_net(self):
+        return self.selenium.find_element(*self._private_net_loc)
+
+    @property
+    def floating_ip_net(self):
+        return self.selenium.find_element(*self._floating_ip_net_loc)
+
+    @property
+    def floating_ip_net_gateway(self):
+        return self.selenium.find_element(*self._floating_ip_net_gateway_loc)
+
+    @property
+    def admin_password(self):
+        return self.selenium.find_element(*self._admin_password_loc)
+
+    @property
+    def confirm_admin_password(self):
+        return self.selenium.find_element(*self._confirm_admin_password_loc)
+
+    @property
+    def external_ceph_storage(self):
+        return self.selenium.find_element(*self._external_ceph_storage_loc)
+
+    # Ceph storage properties
+    @property
+    def ceph_external_mon_host(self):
+        return self.selenium.find_element(*self._ceph_external_mon_host_loc)
+
+    @property
+    def ceph_cluster_fsid(self):
+        return self.selenium.find_element(*self._ceph_cluster_fsid_loc)
+
+    @property
+    def ceph_client_username(self):
+        return self.selenium.find_element(*self._ceph_client_username_loc)
+
+    @property
+    def ceph_client_key(self):
+        return self.selenium.find_element(*self._ceph_client_key_loc)
+
+    @property
+    def nova_rbd_pool_name(self):
+        return self.selenium.find_element(*self._nova_rbd_pool_name_loc)
+
+    @property
+    def cinder_rbd_pool_name(self):
+        return self.selenium.find_element(*self._cinder_rbd_pool_name_loc)
+
+    @property
+    def glance_rbd_pool_name(self):
+        return self.selenium.find_element(*self._glance_rbd_pool_name_loc)
+
+    ###########
+    # actions #
+    ###########
```

Comment:
How about adding 'page element' validation at the actions level before performing actions?